backend/src/auth/dto/login.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class LoginDto {
+  @IsNotEmpty()
+  @IsEmail()
+  email: string;
+
+  @IsNotEmpty()
+  @IsString()
+  hash: string;

```

Comment:
This "hash" and the one inside update.dto.ts might be better named as "password". Since we do the hashing operation inside the backend server. What we receive from the frontend is a "non-hased" password.